src/main/java/de/siegmar/logbackgelf/GelfMessage.java

```diff
@@ -29,13 +27,15 @@
 import java.util.Map;
 import java.util.Objects;
 
+import static java.nio.charset.StandardCharsets.*;
+
 /**
  * Class for GELF 1.1 format representation.
  */
 public class GelfMessage {
 
     private static final String VERSION = "1.1";
-    private static final int INITIAL_SIZE = 256;
+    private static final int INITIAL_SIZE = 502;
```

Comment:
Why? Is 256 to low for average use? Did you mean 512 instead?

Reply:
No. Default packet size is 502 (see your UDP sources).
We have long logs, so usually we fill the whole packet, so it makes sense to make less byte[] reallocations. Not critical, but also no harm. 256 looks like completely abstract number (2⁸) too.